src/bot/conversation.py

```diff
@@ -5,79 +5,81 @@
 from openai import OpenAI
 import os
 from dotenv import load_dotenv
-from src.repository.vocabulary import get_all_words
+from src.repository.vocabulary import get_all_words, get_or_create_user_id_in_DB
+
 
 load_dotenv()
 openai.api_key = os.getenv("OPENAI_API_KEY")
 client = OpenAI()
-user_id = os.getenv("USER_ID")
-
-all_words = get_all_words(user_id)
-"""extract all words of a dictionary - for now i have it locally"""
-
-
-german_words = []
-"""array of all german words in the dictionary"""
-for i in range(len(all_words)):
-    german_words.append(str(all_words[i][1]))
-german_words = str(german_words)
+chat_history = []
```

Comment:
I'm still concerned about that. Why is the chat history defined globally? Doesn't it affect other users? If you have a define globally here I don't see that you use it somewhere else. If you ask me, this can go away.